database/models/slide.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Slide extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      Slide.hasOne(models.Organization, { foreignKey: 'organizationId' });
```

Comment:
se tiene que usar el metodo `belongsTo`, ya que `hasOne` es para relaciones One to One ([fuente](https://sequelize.org/docs/v6/core-concepts/assocs/#implementation-1))

Reply:
Y aca 
```
      Slide.belongsTo(models.Organization, {
        foreignKey:'organizationId'
      })
    }
```